src/main/io/osd.c

```diff
@@ -2543,6 +2556,20 @@ static void osdUpdateStats(void)
         stats.min_rssi = value;
 
     stats.max_altitude = MAX(stats.max_altitude, osdGetAltitude());
+
+    if (osdItemIsVisible((int)OSD_GFORCE)) {
+        //osdUpdateStats is running at 5Hz, that is quite slow to catch true extremum g-force
```

Comment:
I think a proper way to implement this would be to allow IMU to calculate some acceleration stats i.e. average/peak acceleration over last second etc.

Reply:
My intention was to make it as lightweight as possible, but I am not the one to discuss how to implement, since I learn iNav since only few days :)